packages/jest-runtime/src/transform.js

```diff
@@ -159,7 +160,10 @@ module.exports = (
       'jest-transform-cache-' + config.name,
       VERSION,
     );
-    const cacheKey = getCacheKey(preprocessor, content, filename, config);
+    const cacheKey =
+      getCacheKey(preprocessor, content, filename, config) +
```

Comment:
i don't understand why it's needed.
`config` already contains `collectCoverage` and should result in a different cache path.
did `insturment` and `collectCoverage` diverge somewhere in the codebase?